lox_services/persistence/database/push_invoices_data.py

```diff
@@ -58,16 +106,11 @@ def push_run_to_database(
             invoice_path,
             infer_datetime_format=date_format,
             header=0,
-            dtype={"postal_code_reciever": str},
-        )
-        if df_invoice.shape[0] > 0:
-            # If some columns are missing, add the columns with None values
-            for column in dtypes_invoices:
-                if column not in df_invoice.columns:
-                    df_invoice.loc[:, column] = None
+            dtype={"postal_code_reciever": "string[pyarrow]"},
```

Comment:
postal_code_reciever doesn't exist
why postal_code_sender not also included?

Reply:
Typo fixed.